primitives/hp-dispatch/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "hp-dispatch"
+version = "0.1.0"
+license = "Apache-2.0"
+edition.workspace = true
+authors.workspace = true
+repository.workspace = true
+homepage.workspace = true
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+log = "0.4.20"
+binary-merkle-tree = { workspace = true }
+ismp = { workspace = true }
+
+codec = { workspace = true }
+scale-info = { workspace = true, features = ["derive"] }
+
+frame-support = { workspace = true }
+frame-system = { workspace = true }
+frame-benchmarking = { workspace = true, optional = true }
+sp-core = { workspace = true }
+sp-std = { workspace = true }
+sp-runtime = { workspace = true }
+
+hp-on-proof-verified = { workspace = true }
+hex-literal = { workspace = true }
+
+[dev-dependencies]
+pallet-balances = { workspace = true, features = ["std"] }
+sp-io = { workspace = true, features = ["std"] }
+rstest = { workspace = true }
+
+[features]
+runtime-benchmarks = [
```

Comment:
why we need features `runtime-benchmarks` and `try-runtime` here ?

Reply:
Maybe we don't .... I never though about it deeply: maybe if you don't use locally you don't need define them... No that's false in general: if you have a dependency that need `try-runtime` to use _try runtime_ and this pallet is the only one that use it, maybe you should define the feature just to forward it: we already have at least an example of it (for another feature). But in general I don't know how to handle it correctly.